packages/arb-token-bridge-ui/src/hooks/useNativeCurrency.ts

```diff
@@ -16,6 +16,8 @@ export type NativeCurrencyBase = {
 
 export type NativeCurrencyEther = NativeCurrencyBase & {
   isCustom: false
+  address?: undefined
```

Comment:
These make typescript happy where it would be required to check for `isERC20Bridge` all the time.